### DISCUSSION TITLE
🐛 Only creating temporary PDF files?

### DISCUSSION THREAD

**Discussion by Karlstens** [condorheroblog/vitepress-export-pdf#14] [General]
Hey all, this vitepress PDF package looks fantastic - I can see a lot of work has gone into it. 

I've only recently started to explore vitepress, and soon we identified a need to save our docs into PDF files, so bear-with-me that I may be new/ignorant. 

Firstly, I've tested my own markdown project to find that I could only generate temporary PDF files with a hashed(?) name under .\docs\.vitepress\.temp\pdf\

Each of the temporary PDF files created looked to have rendered correctly (_although I'm not sure yet regarding if there's any ability to merge such temp PDF files into one file_) - however, I was expecting a finalised PDF file to be created at the outDir. The directory itself is created but it remains empty - and my VS Code terminal doesn't return an error, other than terminating after the last "Generating" entry from either docs:outFile or docs:outDir. 

Thinking that it may be my own project that's causing issues, I then downloaded and isolated examples\vitepress-docs into its own project and repeated the task. 

After running docs:dev, docs:build, docs:serve (which I added to scripts), they all look as they should. And again, I then attempt docs:outFile / docs:oufDir - but the same problem persists, in that temporary PDF files are created, but nothing beyond that - and I'm expecting named PDF files to appear in the outDir. 

Am I missing a step? Or is this a bug?

**Comment by condorheroblog** [comment]
Can you run this command to show me your environment?

```bash
"scripts": {
	"systemInfo": "press-export-pdf info"
}
```

**Comment by siarhei-chekan** [comment]
I have exactly the same problem.  
In terminal I have an error  message: At least two PDF files.
My environment is:
System:
    OS: Windows 10 10.0.18363
    CPU: (4) x64 Intel(R) Core(TM) i5-2310 CPU @ 2.90GHz
    Memory: 6.93 GB / 15.89 GB
  Binaries:
    Node: 20.11.0 - C:\Program Files\nodejs\node.EXE
    Yarn: Not Found
    npm: 10.2.4 - C:\Program Files\nodejs\npm.CMD
  Utilities:
    Git: 2.34.0.
  Browsers:
    Chrome: 121.0.6167.85
    Edge: Spartan (44.18362.1593.0)
  npmPackages:
    vitepress: ^1.0.0-rc.4 => 1.0.0-rc.40

**Comment by condorheroblog** [comment]
The bug has been fixed, please install the following version

https://github.com/condorheroblog/vitepress-export-pdf/releases/tag/v1.0.0
